### ISSUE TITLE
Custom config not taken into account

### ISSUE THREAD

**Issue by pecuchet** [picocms/Pico#243]
The `get_config` method of `Pico` includes the config file with `$this->config = @include_once(ROOT_DIR . 'config.php');` after which `$this->config` should be an array to be merged with the defaults. I could be mistaken but `include_once` only returns true of false, doesn't it?

In my case at least the following always falls back to $defaults:
```
if (is_array($this->config)) {
    $this->config = array_merge($defaults, $this->config);
} else {
    $this->config = $defaults;
}
```

**Comment by theshka** [comment]
IF (this instance of $config is an array) { merge the array with the defaults } ELSE { this instance of $config is equal to the default array. } is proper. My custom configuration is taken in to account. Did you rename the `config.php.template` file to `config.php` and uncomment the lines you are trying to affect?

**Comment by pecuchet** [comment]
Strange, I just reverted the changes I made on the get_config method of Pico, just to double check. The problem is the same: when I have $this->config = @include_once(ROOT_DIR . 'config.php') (line 203) $this->config is not assigned with the array values in config.php, but merely 'true'. After that the first error I get is "file_get_contents(content-sample/404.md): failed to open stream", which is logical since in my config.php the content folder is not 'content-sample' and there is no content-sample folder in the root.

The if-else is indeed ok, but for me include_once only returns a bool (as expected). Here is the simple change I made on the get_config method:

```
@include_once(ROOT_DIR . 'config.php');

$defaults = array(
    // left as is
);

if (is_array($config)) {
    $this->config = array_merge($defaults, $config);
} else {
    $this->config = $defaults;
}
```
